cfgov/scripts/initial_test_data.py

```diff
@@ -39,7 +39,7 @@ def run():
     site_root = HomePage.objects.filter(title='CFGOV')
     if not site_root:
         root = Page.objects.first()
-        site_root = HomePage(title='CFGOV', slug='home', depth=2, owner=admin_user)
+        site_root = HomePage(title='CFGOV', slug='home-page', depth=2, owner=admin_user)
```

Comment:
@vccabral That seems a little out of scope for this PR (which is already pretty big) -  but curious why you think that?